nym-vpn-lib/src/platform/android.rs

```diff
@@ -173,11 +180,94 @@ pub extern "system" fn Java_net_nymtech_vpn_NymVpnService_defaultTunConfig<'env>
     env: JNIEnv<'env>,
     _this: JObject<'_>,
 ) -> JObject<'env> {
+    use jnix::IntoJava;
     let env = JnixEnv::from(env);
-
     TunConfig::default().into_java(&env).forget()
 }
 
+// ideally we use IntoJava trait from jnix, quick fix for now
+impl<'env> IntoJava<'env> for String {
```

Comment:
suggestion: usually the interface (trait) comes before the concrete implementation